ddtrace/propagation/http.py

```diff
@@ -988,6 +990,24 @@ def _extract_configured_contexts_avail(normalized_headers):
                 styles_w_ctx.append(prop_style)
         return contexts, styles_w_ctx
 
+    @staticmethod
+    def _context_to_span_link(context: Context, style: str, reason: str) -> Optional[SpanLink]:
+        # encoding expects at least trace_id and span_id
+        if context.span_id and context.trace_id:
```

Comment:
What do other languages do here? We should ensure this behavior aligned across tracers. Either we need to allow span links with Trace_ID=0 and SPAN_ID=0 (to record conflicting/ignored baggage headers) or we need a well defined way to merge baggage headers in one format (tracecontext) to another (x-datadog headers).

Reply:
Since baggage is evaluated separately, it's always added the final context we decide on, whether it has span links decided through `DD_TRACE_PROPAGATION_STYLE=restart`, or through conflicting incoming contexts. So I don't think this is actually an issue. If a context is formed from a propagation style other than baggage, it'll always have a trace_id at least.